devtools-ci-index.yaml

```diff
@@ -2134,6 +2165,24 @@
             timeout: '30m'
         - '{ci_project}-{git_repo}-fabric8-analytics-pylint':
             git_repo: fabric8-analytics-server
+        - '{ci_project}-{git_repo}-fabric8-hdd':
+            git_organization: fabric8-hdd
+            git_repo: supervisor
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '30m'
+            image_name: 'fabric8-hdd/openshift-hdd-supervisor'
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8-hdd
+            git_repo: supervisor
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-hdd # To Be created
+            deployment_units: 'supervisor'
+            deployment_configs: ''
```

Comment:
Why is this an empty string?

Reply:
I wanted to know if this is correct:
https://github.com/fabric8-hdd/almighty-jobs/pull/1/files/6fb01fb71b132f0a41a236a9c9c8f6ef68da7524#diff-a0fd432ddaac45438209cf6e0a7298c0R2181 maps to https://github.com/fabric8-hdd/saas-hdd/blob/master/hdd-services/supervisor.yaml#L4
and
https://github.com/fabric8-hdd/almighty-jobs/pull/1/files/6fb01fb71b132f0a41a236a9c9c8f6ef68da7524#diff-a0fd432ddaac45438209cf6e0a7298c0R2182 maps to https://github.com/fabric8-hdd/supervisor/blob/master/openshift/template.yaml#L85